app/models/course_queue.rb

```diff
@@ -6,6 +6,9 @@ class CourseQueue < ApplicationRecord
 
   def request(requester:, description:, location:, group:)
     self.with_lock do
+      if self.course.course_queue_entries.where(resolved_at: nil).where(requester: requester).count > 0
+          raise "Limit one open request per user"
+      end
```

Comment:
👋 thanks for the PR! Is this check (below) still necessary?

https://github.com/mterwill/office-hours-help-queue/blob/633bef05ae63750d4820dd41ddbbabb5bd083a35/app/models/course_queue.rb#L10-L12

Reply:
You're right! Removed in most recent commit.